frontend/src/app/services.ts

```diff
@@ -320,7 +380,13 @@ const mutationApi = hitasApi.injectEndpoints({
                 body: data,
                 headers: {"Content-type": "application/json; charset=UTF-8"},
             }),
-            invalidatesTags: (result, error, arg) => [{type: "Apartment"}, {type: "Index", id: "LIST"}],
+            invalidatesTags: (result, error, arg) => [
+                {type: "Apartment"},
+                {
+                    type: "Index",
+                    id: "LIST",
+                },
+            ],
```

Comment:
Why reformat all of these `invalidatesTags` to a more expanded, longer format? I don't see any actual changes with them 🤨

Reply:
WebStorm config fixed, this shouldn't happen anymore even by mistake.